app/src/main/java/de/westnordost/streetcomplete/util/logs/Logger.kt

```diff
@@ -0,0 +1,24 @@
+package de.westnordost.streetcomplete.util.logs
+
+interface Logger {
+    /** Send VERBOSE log message */
+    fun v(tag: String, message: String)
+
+    /** Send DEBUG log message */
+    fun d(tag: String, message: String)
+
+    /** Send INFO log message */
+    fun i(tag: String, message: String)
+
+    /** Send WARNING log message */
+    fun w(tag: String, message: String)
+
+    /** Send WARNING log message and exception */
+    fun w(tag: String, message: String, e: Throwable)
+
+    /** Send ERROR log message */
+    fun e(tag: String, message: String)
+
+    /** Send ERROR log message and exception */
+    fun e(tag: String, message: String, e: Throwable)
```

Comment:
This builds upon @westnordost's [suggestion](https://github.com/streetcomplete/StreetComplete/pull/5335/files#r1376893189) but also uses KDoc's [links to elements](https://kotlinlang.org/docs/kotlin-doc.html#links-to-elements):

```suggestion
    /** Send VERBOSE log [message] */
    fun v(tag: String, message: String)

    /** Send DEBUG log [message] */
    fun d(tag: String, message: String)

    /** Send INFO log [message] */
    fun i(tag: String, message: String)

    /** Send WARNING log [message] with optional [exception] */
    fun w(tag: String, message: String, exception: Throwable? = null)

    /** Send ERROR log message with optional [exception] */
    fun e(tag: String, message: String, exception: Throwable? = null)
```

Reply:
I'm pretty new to Kotlin, so thanks for the tip! 😄